src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,41 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void contains(int number) {
+        assertThat(numbers.contains(number)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource({"1,true", "2,true", "3,true", "4,false", "5,false"})
+    void contains_whenNotExistThenFalse(int number, boolean result) {
+        assertThat(numbers.contains(number)).isEqualTo(result);
+    }
```

Comment:
반복된 테스트 케이스를 돌렸을 때 어떤 값이 들어갔고 어떤 결과가 나와야하는지 
파라미터에 대한 설명도 같이 포함되면 좋을 것 같아요 

현재는 테스트 코드가 간단하지만 복잡한 경우에는 테스트 결과만 보고 파악하기에는 어려울 것 같아요
<img width="248" alt="image" src="https://user-images.githubusercontent.com/36617783/229539567-dc38eda8-85db-4256-97e8-68f796f32ca1.png">

```suggestion
    @ParameterizedTest(name = "{0} 포함 여부: {1}")
    @CsvSource({"1,true", "2,true", "3,true", "4,false", "5,false"})
    void contains_whenNotExistThenFalse(int number, boolean result) {
        assertThat(numbers.contains(number)).isEqualTo(result);
    }
```

수정해 보시고 결과를 한 번 확인해 보시죠!